.github/workflows/notarize-immudb4net.yml

```diff
@@ -1,17 +1,16 @@
 name: Notarize immudb4net
 
 on:
-  pull_request:
+  push:
```

Comment:
Makes sense to notarize also on every tag

Reply:
I tested it and the tag doesn't change the commit, so it's not needed. If I put a tag filter, then it will notarize only when the tag is present...